roles/openshift_examples/examples-sync.sh

```diff
@@ -9,7 +9,7 @@ XPAAS_VERSION=ose-v1.3.3
 ORIGIN_VERSION=${1:-v1.3}
 EXAMPLES_BASE=$(pwd)/files/examples/${ORIGIN_VERSION}
 find ${EXAMPLES_BASE} -name '*.json' -delete
-find ${EXAMPLES_BASE} -name '*.yaml' -delete
+find ${EXAMPLES_BASE} -name '*.yaml' -delete -exclude registry-console.json
```

Comment:
Yeah, that may be a good way to solve the problem. I'm still hopeful that we'll get an upstream repo to pull from and we'll just be downstream.